client/src/partials/primary_cash/expense/purchase.html

```diff
@@ -70,13 +70,13 @@
     </div>
 
     <div class="col-xs-7">
-      <div class="panel square" ng-class="{'panel-success': session.selected, 'panel-default' : !session.selected}">
+      <div class="panel square" ng-class="{'panel-success': PurchaseCtrl.session.selected, 'panel-default' : !PurchaseCtrl.session.selected}">
```

Comment:
The class .square doesn't do anything in bhima (and probably never will).  you can remove it while you are here.